write-fonts/src/graph.rs

```diff
@@ -1169,7 +1169,10 @@ impl Graph {
 
     #[cfg(feature = "dot2")]
     pub(crate) fn write_graph_viz(&self, path: impl AsRef<std::path::Path>) -> std::io::Result<()> {
-        graphviz::GraphVizGraph::from_graph(self).write_to_file(path)
+        // if this is set then we prune the generated graph
+        const PRUNE_GRAPH_ENV_VAR: &str = "FONTC_PRUNE_GRAPH";
```

Comment:
(bikeshedding) I feel the var name ties too much write-fonts with fontc.. conceivable another tool might find it useful.. maybe WRITE_FONTS_PRUNE_GRAPH

Reply:
I was thinking that `WRITE_` was confusing as a prefix, as it's a verb. Given that we own both projects, and that this is undocumented/private API, I thiiiink I prefer this version? open to more ideas though..

Reply:
`FONTATIONS_PRUNE_GRAPH`

Reply:
but it's fine like that, since it's undocumented/private api anyway